### PR TITLE
Add endpoint to add a user

### PR DESCRIPTION
**Pull request by Mariana0711**
What was done:
- Add the `addPerson` function (endpoint: `/onboarding/user`) to create a new user, it'll receive an `userForm` of `PersonUpdateRequest` type.
- The `updatePerson` function will return an object so on the flexbase-web side I can update the user updated with the object returned.
- Add tests for the `addPerson` function.

### CODE REVIEW

File: src/clients/FlexbaseClient.Person.ts [hunk 1]
```diff
@@ -27,12 +28,12 @@ interface PersonResponse extends FlexbaseResponse {
...
+    async getEmployees(): Promise<PersonUpdateRequest[]> {
         try {
           return await this.client
           .url('/user')
           .get()
-          .json<Person[]>();
+          .json();
 
         } catch (error) {
           console.error('ALL USERS ERROR', error);
```
Comment (tophercf) [inline src/clients/FlexbaseClient.Person.ts]: is the pattern to return `null` on error?

File: src/clients/FlexbaseClient.Person.ts [hunk 2]
```diff
@@ -60,7 +61,20 @@ export class FlexbaseClientPerson extends FlexbaseClientBase {
...
-    async updatePerson(userId: string, person: PersonUpdate): Promise<boolean> {
+    async addPerson(userForm: PersonUpdateRequest): Promise<Person | null> {
+        try {
+          const result = await this.client
+            .url('/onboarding/user')
+            .post(userForm)
+            .json();
+          return result.newUser;
+        } catch (error) {
+          console.error('SAVE USER ERROR', error);
```
Comment (tophercf) [inline src/clients/FlexbaseClient.Person.ts]: should we be using console.error() or this.logger.error(`some message`) ?

File: src/clients/FlexbaseClient.Person.ts (hunk 1 above)
Comment (Mariana0711) [reply src/clients/FlexbaseClient.Person.ts]: Yes, it is

File: src/clients/FlexbaseClient.Person.ts (hunk 2 above)
Comment (Mariana0711) [reply src/clients/FlexbaseClient.Person.ts]: You're right!... Fixing it

**Review by tophercf — DISMISSED** [review]

**Review by xXAndrexPereXx — DISMISSED** [review]

**Review by Dayana0721 — APPROVED** [review]
🤓

**Review by xXAndrexPereXx — APPROVED** [review]

**Pull request merged**